.github/workflows/unit_tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - name: 'Unit testing'
         run: |
           testenv\Scripts\Activate.ps1
-          pytest -n 6 --dist loadfile -v --cov=pyaedt  --cov-report=xml --junitxml=junit/test-results.xml --cov-report=html _unittest -p no:faulthandler
+          pytest -n 6 --dist loadfile -v --cov=pyaedt  --cov-report=xml --junitxml=junit/test-results.xml --cov-report=html _unittest -o faulthandler_timeout=1800
```

Comment:
This should fix most of the timeout issue that we experimented recently. Thanks @maxcapodi78

Reply:
yes I hope so